app/src/main/java/com/rootstrap/android/ui/products_list/ProductListViewModel.kt

```diff
@@ -0,0 +1,28 @@
+package com.rootstrap.android.ui.products_list
+
+import androidx.lifecycle.viewModelScope
+import com.rootstrap.android.ui.base.BaseViewModel
+import com.rootstrap.android.ui.base.UiState
+import com.rootstrap.domain.Product
+import com.rootstrap.usecases.GetProducts
+import kotlinx.coroutines.launch
+
+class ProductListViewModel(private val getProducts: GetProducts) :
+    BaseViewModel<ProductListUiState>(ProductListUiState()) {
+    fun load() {
+        viewModelScope.launch {
+            val products = getProducts()
```

Comment:
If in the future, this will connect with an API, remember to change its context to `IO`

Reply:
Sure, thanks!